packages/website/docs/with-instantsearch.md

```diff
@@ -0,0 +1,51 @@
+---
+id: with-instantsearch
+title: Autocomplete with InstantSearch
+---
+
+Learn how to allow users to search from anywhere on your website, using Autocomplete with InstantSearch.
+
+## Why use Autocomplete if I've got InstantSearch
```

Comment:
I feel like some of this content should be used in the [**Introduction**](https://autocomplete.algolia.com/docs/introduction). This is not directly related to InstantSearch but we can refer to it here.

Reply:
Agree -- especially if we talk generically about incorporating Autocomplete into an existing generic search -> search results page UX.